egs2/librispeech/ssl1/conf/deepspeed.json

```diff
@@ -0,0 +1,47 @@
+{
+    "train_micro_batch_size_per_gpu": 1,
+      "gradient_accumulation_steps": 1,
+      "gradient_clipping": 5.0,
+      "bf16": {
```

Comment:
BF16 is only suitable for Ampere and above. Add a comment here?

Reply:
I put it in the readme instead. I dont think json supports comments natively.